src/data/markdown/docs/07 extensions/02 Explanations/extensions-registry.md

```diff
@@ -0,0 +1,61 @@
+---
+title: About the Extensions Registry
+excerpt: Reasons for the registry and what is required to be included.
+hideFromSidebar: false
+---
+
+Our desire is to provide the best developer experience when using k6.
+This extends to the extensions ecosystem as well.
+The adaptability provided by k6 extensions opens a wide-array of potential use cases.
+
+The need for a well-maintained, curated listing of extensions is a necessity.
+Our pledge to the community will be to make our best attempt to ensure those projects listed meet certain standards.
+While we cannot guarantee the quality of community-provided extensions, we _can_ aid the evaluation by requiring certain consistencies.
+
+Did you create an extension and want to share it with your fellow k6 users?
+We'd love to spread word of this new feature adding to our [registry](/extensions/get-started/explore/) of available extensions.
```

Comment:
If this is to be in the doc, it seems like it should be the first paragraph, with an added sentence about requirements:

Did you create an extension and want to share it with your fellow k6 users?
We'd love to spread word of this new feature adding to our [registry](/extensions/get-started/explore/) of available extensions.
However, before an extension is added to the registry, we must ensure that it meets our quality standards.

Reply:
@MattDodsonEnglish, I'll move this up and add the extra sentence. I do worry about the "meets our quality standards" part as we can't really enforce how well an extension is written. 

Curious if you'd be good with "meets our minimum requirements"?